sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventDataBatchIntegrationTest.java

```diff
@@ -52,7 +52,9 @@ protected void beforeTest() {
             .shareConnection()
             .consumerGroup(EventHubClientBuilder.DEFAULT_CONSUMER_GROUP_NAME)
             .prefetchCount(EventHubClientBuilder.DEFAULT_PREFETCH_COUNT);
-        producer = toClose(builder.buildAsyncProducerClient());
+        producer = toClose(builder
+            .retryOptions(RETRY_OPTIONS)
```

Comment:
this supposes to fix transient timeout (sets timeout to 3 sec instead of 1 min) so that tests have a chance to retry things